### PR TITLE
Fix max allowed lengths for prompts larger than 4096

### PR DESCRIPTION
**Pull request by ani300**
This takes cares of an oversight that prevented prompts longer than 4k from being run with generate()

### CODE REVIEW

File: aiu_fms_testing_utils/utils/paged.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def adjust_inputs_to_batch(input_ids: torch.Tensor, **extra_kwargs):
 def generate(
     model: Union[Callable, torch.nn.Module],
     input_ids: torch.Tensor,
+    max_seq_len: int = 4096,
```
Comment (JRosenkranz) [inline aiu_fms_testing_utils/utils/paged.py:30]: Do we anticipate this getting used? or is it just a placeholder to grab the max_seq_len param if passed to fms generate?

Comment (ani300) [reply aiu_fms_testing_utils/utils/paged.py:30]: this is to match the signature between both generate() calls and avoid errors yes

File: aiu_fms_testing_utils/utils/__init__.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ def warmup_model(
         generate(
             model,
             _warmup_input_ids,
+            max_seq_len=_warmup_input_ids.shape[1] + max_new_tokens,
```
Comment (JRosenkranz) [inline aiu_fms_testing_utils/utils/__init__.py:56]: could we just add this to the `attention_specific_kwargs` in line 32? this way we don't need to change the signature of paged generate.

File: aiu_fms_testing_utils/utils/paged.py (hunk 1 above)
Comment (JRosenkranz) [reply aiu_fms_testing_utils/utils/paged.py:30]: We may just want this to be part of the attention_specific_kwargs in the inference and warmup

Comment (ani300) [reply aiu_fms_testing_utils/utils/paged.py]: done
